nixos/modules/hardware/video/nvidia.nix

```diff
@@ -296,7 +296,7 @@ in
               message = "You cannot configure both X11 and Data Center drivers at the same time.";
             }
             {
-              assertion = cfg.open != null;
+              assertion = cfg.open != null || cfg.datacenter.enable;
```

Comment:
This will unconditionally cause an error if `hardware.nvidia.datacenter.enable` is `true`. That can't be the intended effect.

Reply:
Nvm, I read the assertion the wrong way around